### ISSUE TITLE
 pyinstaller --name=project-name path/to/my/main.py invalid syntax

### ISSUE THREAD

**Issue by mattshirtliffe** [pyinstaller/pyinstaller#1225]
I ran pyinstaller --name=project-name path/to/my/main.py and got the message below
```
2351 ERROR: invalid syntax (__init__.py, line 855)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/PyInstaller-2.1.1dev_67610f2-py2.7.egg/PyInstaller/depend/owner.py", line 110, in getmod
    co = compile(stuff.replace("\r\n", "\n"), py[0], 'exec')
  File "xlrd3/__init__.py", line 855
    print("mem: %s, base: %d, len: %d" % (type(self.mem), self.base, self.stream_len), file=self.logfile)
                                                                                           ^
SyntaxError: invalid syntax
```

**Comment by htgoebel** [comment]
This is an error in your file "xlrd3/init.py", line 855, which seams to be a python3 file.

**Comment by mattshirtliffe** [comment]
I have just set up Vagrant so I have a clean machine with clean python. I have just ran  python --version and it says I have version Python 2.7.6. I'm trying to use pyinstaller to package my a flask application stand-alone executables if that helps. Here are a list of my modules I'm using if that helps. 

BeautifulSoup
bson
celery
Crypto
flask
jinja2
logbook
models
pyftpdlib
pymongo
redis
rq
tablib
tornado
werkzeug
wtforms
mongoengine
